src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -802,17 +802,27 @@ def get_num_units(self) -> int:
         return self.sorting.get_num_units()
 
     ## extensions zone
-
-    def compute(self, input, save=True, **kwargs):
+    def compute(self, input, save=True, extension_params=None, **kwargs):
         """
         Compute one extension or several extensiosn.
         Internally calls compute_one_extension() or compute_several_extensions() depending on the input type.
 
         Parameters
         ----------
         input: str or dict or list
-            If the input is a string then computes one extension with compute_one_extension(extension_name=input, ...)
-            If the input is a dict then compute several extensions with compute_several_extensions(extensions=input)
+            The extesions to compute, which can be passed as:
+
+            * a string: compute one extension. Additional parameters can be passed as key word arguments.
+            * a dict: compute several extensions. The keys are the extension names and the values are the extension parameters.
+            * a list: compute several extensions. The list contains the extension names. Additional parameters can be passed with the extension_params
+              argument.
```

Comment:
I'm not sure if this should go here or not, but how does one pass default params? If I'm the user reading the docstring I'm not sure if it is clear. Maybe it is, but for example if I'm doing a dict: what if I do {'random_spikes': []} instead of {'random_spikes': {}}? or for the list I might assume that if I specify for one I have to specify for all. Maybe we need this in the actual docs rather than here, but something we should think about.